bubble-dns/include

```diff
@@ -32,6 +33,7 @@ updateBubbleDNS() {
     echo "Error: cannot find configuration 'domain secret'"
```

Comment:
Maybe use `color e` or `color w`? (same in lines 18, 28)

Reply:
this is an error: then `color e` makes sense. It will also send message to `/std/err` 

> 
```suggestion
    color e "Cannot find configuration 'domain secret'"
```
    color e "Cannot find configuration 'domain secret'"
````

> Error: ` prefix in this case is redundant
```suggestion
    color e "Cannot find configuration 'domain secret'"
```

Reply:
Btw... do we need `bubble-dns/include` script at all? Can team confirm if this is true: `aws` was the last extension which was using this script?